feature/session/src/main/res/layout/fragment_speaker.xml

```diff
@@ -1,195 +1,196 @@
+                        app:circleCrop="@{true}"
+                        app:imageUrl="@{speaker.imageUrl}"
+                        app:layout_constraintEnd_toEndOf="parent"
+                        app:layout_constraintTop_toTopOf="parent"
+                        app:placeHolder="@{@drawable/ic_person_outline_black_24dp}"
+                        app:placeHolderTint="@{@color/gray2}"
+                        tools:src="@tools:sample/avatars"
+                />
 
                 <ImageView
-                    android:id="@+id/divider1"
-                    android:layout_width="0dp"
-                    android:layout_height="4dp"
-                    android:layout_marginTop="56dp"
-                    android:contentDescription="@null"
-                    android:layerType="software"
-                    android:src="@drawable/shape_dashed_line"
-                    app:layout_constraintEnd_toEndOf="parent"
-                    app:layout_constraintStart_toStartOf="parent"
-                    app:layout_constraintTop_toBottomOf="@id/speaker_tag_line"
-                    />
+                        android:id="@+id/divider1"
+                        android:layout_width="0dp"
+                        android:layout_height="4dp"
+                        android:layout_marginTop="56dp"
+                        android:contentDescription="@null"
+                        android:layerType="software"
+                        android:src="@drawable/shape_dashed_line"
+                        app:layout_constraintEnd_toEndOf="parent"
+                        app:layout_constraintStart_toStartOf="parent"
+                        app:layout_constraintTop_toBottomOf="@id/speaker_tag_line"
+                />
 
                 <androidx.emoji.widget.EmojiTextView
-                    android:id="@+id/speaker_description"
-                    android:layout_width="0dp"
-                    android:layout_height="wrap_content"
-                    android:layout_marginEnd="16dp"
-                    android:layout_marginTop="16dp"
-                    android:ellipsize="end"
-                    android:includeFontPadding="false"
-                    android:paddingBottom="2dp"
-                    android:text="@{speaker.bio}"
-                    android:textAlignment="viewStart"
-                    app:layout_constraintEnd_toEndOf="parent"
-                    app:layout_constraintStart_toStartOf="@id/speaker_name"
-                    app:layout_constraintTop_toBottomOf="@id/divider1"
-                    tools:text="Material is the metaphor A material metaphor is the unifying theory of a rationalized space and a system of motion. The material is grounded in tactile reality, inspired by the study of paper and ink, yet technologically advanced and open to imagination and magic."
-                    />
+                        android:id="@+id/speaker_description"
+                        android:autoLink="web"
```

Comment:
autolink does not always enable link clickable. For example, the TextView allows selecting its text, enabling link clickable would be skipped so the link would not work correctly.
Could you please set LinkMovementMethod expressly?

Reply:
ok. I'll fix it.

Reply:
@jmatsu fixed